tests/test_display_options_integration.py

```
@@ -138,10 +138,13 @@
     ],
 )
 async def test_display_options_state_render(
-    display_option: str, expected_state: str, mock_hass, patch_entity_registry, monkeypatch
-):
+    display_option: str,
+    expected_state: str,
+    mock_hass: MagicMock,
+    patch_entity_registry: object,
```

Ruff (0.15.12)

[warning] Unused function argument: `patch_entity_registry`

(ARG001)